worker/worker.go

```diff
@@ -654,6 +654,17 @@ func (w *worker) objectsKeyHandlerPUT(jc jape.Context) {
 	}
 	rs := up.RedundancySettings
 
+	// allow overriding the redundancy settings
+	if jc.Check("couldn't decode 'minshards' parameter", jc.DecodeForm("minshards", &rs.MinShards)) != nil {
```

Comment:
`DecodeForm` already writes an error on failure, you don't need to `Check` it.

(I suppose this is one of the downsides of having these functions return the `error`, instead of a `bool`)

Reply:
Oh, silly oversight, thanks for catching it.